### PR TITLE
Revamp Gora Site

### PR DESCRIPTION
**Pull request by avivkeller**
This pull request revamps the layout of the Gora site using Bootstrap 5.3 over 2.3.1

# Changes
- Fix Navbar dropdowns
- Modify navbar image size
- Remove searchbar
- Replace hero with plain text
- Change 'News' to a accordion layout
- Update mailing list to a single table
- Update people to a table
- Change 'Project Champion (PC)' to 'Project Champion (👑)'
- Add missing links to people page (To the best of my ability)
- Add `.gitignore` for python env, cache, and pelican output
- And more

### CODE REVIEW

**Review by lewismc — COMMENTED** [review]
Hi @RedYetiDev wow thank you so much for the PR this is a pleasant surprise. 
Can you please update README with guidance on how to build the website locally?
Thank you

**Comment by avivkeller** [comment]
Sure! I added `markdown` the requirements.txt, as it should've been there in the first place, and I redid the README.md file.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+```bash
+pip install -r requirements.txt
+```
+> Note: You may also need to run `pip install pelican[markdown]`
+
+### Building
+4. Utilize Pelican to generate the website.
+
+```bash
+pelican content
```
Comment (lewismc) [inline README.md:37]: I was able to see the website deployed locally with `pelican content -l`.
`stdout` complained about `Cannot load plugin pelican-gfm`
Content was served at http://127.0.0.1:8000 which is really useful

File: README.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+python3 -m venv env
+source env/bin/activate
+```
+
+3. Install the required Python packages using pip.
+
+```bash
+pip install -r requirements.txt
+```
+> Note: You may also need to run `pip install pelican[markdown]`
```
Comment (lewismc) [inline README.md]:
```
(env) ➜  gora-site git:(main) pip install pelican[markdown]
zsh: no matches found: pelican[markdown]
```

**Review by lewismc — CHANGES_REQUESTED** [review]
Please see comments @RedYetiDev

**Comment by lewismc** [comment]
Additionally in the pelican output I see the following
```
current/gora-shims.md - Gora Core Module
id="description" is a duplicate
id="dependency-definition" is a duplicate
id="description" is a duplicate
id="description_1" is a duplicate
id="dependency-definition" is a duplicate
id="dependency-definition_1" is a duplicate
id="description" is a duplicate
id="description_1" is a duplicate
id="description_2" is a duplicate
id="dependency-definition" is a duplicate
id="dependency-definition_1" is a duplicate
id="dependency-definition_2" is a duplicate
```

and

```
current/gora-pig.md - Gora Pig Module
  ToC
    #primitivesimple-types
    #introduction
    #reading-from-datastores
    #overview
    #deleting-elements
    #writing-to-datastores
    #data-models
    #complex-types
    #full-options-for-load
current/gora-core.md - Gora Core Module
id="description" is a duplicate
id="goraproperties" is a duplicate
id="description" is a duplicate
id="description_1" is a duplicate
id="goraproperties" is a duplicate
id="goraproperties_1" is a duplicate
id="description" is a duplicate
id="description_1" is a duplicate
id="description_2" is a duplicate
```

**Comment by lewismc** [comment]
The `Nightly Trunk Javadoc` hyperlink target is broken. I’m honestly not sure what it should be anymore. It looks like the Gora build at https://ci-builds.apache.org/job/Gora/ are… somewhat neglected. 

I think we can remove the navigation to the Gora wikipedia entry at http://en.wikipedia.org/wiki/Apache_Gora

Also the Sonar Analysis navigation can be removed https://analysis.apache.org/dashboard/index/76356

Finally, the `Nightly Builds` navigation can be removed at https://builds.apache.org/view/All/job/gora-trunk/

File: README.md (hunk 2 above)
Comment (avivkeller) [reply README.md]: Depending on the system, will need to escape it in quotes: `pip install "pelican[markdown]"`, I can update the README if need be

File: README.md (hunk 1 above)
Comment (avivkeller) [reply README.md:37]: I know, stdout has been complaining about that from even before my changes, but it didn't cause an issue, so I left it be.

**Comment by avivkeller** [comment]
> ```
> current/gora-shims.md - Gora Core Module
> id="description" is a duplicate
> id="dependency-definition" is a duplicate
> id="description" is a duplicate
> id="description_1" is a duplicate
> id="dependency-definition" is a duplicate
> id="dependency-definition_1" is a duplicate
> id="description" is a duplicate
> id="description_1" is a duplicate
> id="description_2" is a duplicate
> id="dependency-definition" is a duplicate
> id="dependency-definition_1" is a duplicate
> id="dependency-definition_2" is a duplicate
> ```
I didn't create the files that have these issues, but I can try to refactor them if you'd like.

File: README.md (hunk 2 above)
Comment (avivkeller) [reply README.md]: Resolved. Once I resolve the rest, I will squash.

**Comment by avivkeller** [comment]
Okay, @lewismc, I've made the changes requested. Once you confirm there is nothing else, I'll squash the commits for clarity.

File: README.md (hunk 1 above)
Comment (avivkeller) [reply README.md:37]: Marking as resolved as I see no reason for change, if you think otherwise please let me know

**Comment by avivkeller** [comment]
The code has been squashed for ease of future merging.

**Review by lewismc — APPROVED** [review]
+1 excellent job @RedYetiDev

**Comment by avivkeller** [comment]
Thanks!

**Pull request merged**